### ISSUE TITLE
关于capsule_softmax_weight 的计算

### ISSUE THREAD

**Issue by wutengcoding** [THUDM/ComiRec#13]
大佬， capsule_weight = tf.stop_gradient(tf.truncated_normal([get_shape(item_his_emb)[0], self.num_interest, self.seq_len], stddev=1.0))
model.py第153行，按照胶囊网络的结构， capsule_softmax_weight = tf.nn.softmax(capsule_weight, axis=1)  应该是在 seq_len这个维度做softmax才对呀
这样保证低capsule的输入权重相加之和是1， 这块没理解为啥在 num_interest的维度做softmax

**Comment by cenyk1230** [comment]
Hi @wutengcoding , this issue is the same as #2.
